### PR TITLE
Remove unused stuff

### PR DESCRIPTION
**Pull request by chrisrichardson**
Resolves Issue #598

### CODE REVIEW

File: ffcx/codegeneration/lnodes.py [hunk 1]
```diff
@@ -831,7 +831,15 @@ def as_statement(node):
 
 
 def _math_function(op, *args):
-    return MathFunction(op._ufl_handler_name_, args)
+    name = op._ufl_handler_name_
+    dtype = args[0].dtype
+    if name in ("conj", "real") and dtype == DataType.REAL:
```
Comment (michalhabera) [inline ffcx/codegeneration/lnodes.py:836]: This seems unrelated?

Comment (chrisrichardson) [reply ffcx/codegeneration/lnodes.py:836]: Yes just a merge of Matthew s branch. I will fix.
